src/AutomatedCar/Views/CourseDisplayView.xaml

```diff
@@ -64,6 +64,54 @@
 							Fill="{Binding RadarBrush, Mode=OneWay}"
 							Points="{Binding RadarGeometry.Points, Mode=OneWay}" />
 
+                        <Polyline
+							IsVisible="{Binding UltraSoundVisible, Mode=OneWay}"
+							Opacity=".4"
+							Fill="{Binding UltraSoundBrush, Mode=OneWay}"
+							Points="{Binding UltraSoundGeometries[0].Points, Mode=OneWay}" />
+
+                        <Polyline
+							IsVisible="{Binding UltraSoundVisible, Mode=OneWay}"
+							Opacity=".4"
+							Fill="{Binding UltraSoundBrush, Mode=OneWay}"
+							Points="{Binding UltraSoundGeometries[1].Points, Mode=OneWay}" />
+
+                        <Polyline
+							IsVisible="{Binding UltraSoundVisible, Mode=OneWay}"
+							Opacity=".4"
+							Fill="{Binding UltraSoundBrush, Mode=OneWay}"
+							Points="{Binding UltraSoundGeometries[2].Points, Mode=OneWay}" />
+
+                        <Polyline
+							IsVisible="{Binding UltraSoundVisible, Mode=OneWay}"
+							Opacity=".4"
+							Fill="{Binding UltraSoundBrush, Mode=OneWay}"
+							Points="{Binding UltraSoundGeometries[3].Points, Mode=OneWay}" />
+
+                        <Polyline
+							IsVisible="{Binding UltraSoundVisible, Mode=OneWay}"
+							Opacity=".4"
+							Fill="{Binding UltraSoundBrush, Mode=OneWay}"
+							Points="{Binding UltraSoundGeometries[4].Points, Mode=OneWay}" />
+
+                        <Polyline
+							IsVisible="{Binding UltraSoundVisible, Mode=OneWay}"
+							Opacity=".4"
+							Fill="{Binding UltraSoundBrush, Mode=OneWay}"
+							Points="{Binding UltraSoundGeometries[5].Points, Mode=OneWay}" />
+
+                        <Polyline
+							IsVisible="{Binding UltraSoundVisible, Mode=OneWay}"
+							Opacity=".4"
+							Fill="{Binding UltraSoundBrush, Mode=OneWay}"
+							Points="{Binding UltraSoundGeometries[6].Points, Mode=OneWay}" />
+
+                        <Polyline
+							IsVisible="{Binding UltraSoundVisible, Mode=OneWay}"
+							Opacity=".4"
+							Fill="{Binding UltraSoundBrush, Mode=OneWay}"
+							Points="{Binding UltraSoundGeometries[7].Points, Mode=OneWay}" />
+
```

Comment:
Ez nem csak egy háromszög kell legyen?

Reply:
3szog, de nem 1 hanem 8 db sensor van a kocsin emiatt van 8db Polyline.
A probléma pont az hogy nem tudom hogy kéne ezt foreach formában megjeleniteni

Reply:
Á, okés. ez kiesett nekem. :)


Ezt próbáld meg:
<ItemsControl Items="{Binding Geoms}">
    <ItemsControl.ItemTemplate>
        <DataTemplate>
        <Polyline Stroke="blue" StrokeThickness="3" Points="{Binding Points, Mode=OneWay}" ZIndex="85"/>
        </DataTemplate>
    </ItemsControl.ItemTemplate>
</ItemsControl>

https://github.com/SzFMV2020-Osz/AutomatedCar-B/pull/78

Reply:
@zsofiath A fenti pédánál, ha jól értem a müködését akkor a `DataTemplate`-en bellüli rész csak az `Items` `Binding`-on keresztül látható, szóval ha nekem van egy `UltraSoundVisible` propertym ami nem az `Items` scope-ban van hanem azzal egyszinten akkor azt nem lehet belerakni? vagy van más megoldás?

Reply:
Ha jól értem, akkor ez a rész akkor most kollekció kezelésre vár, így nincs kész.

Reply:
Megpróbáltam Összehozni, de nem sikerült. Elvileg ezt így kéne:
```c#
<ItemsControl Items="{Binding UltraSoundGeometries}">
	<ItemsControl.ItemTemplate>
		<DataTemplate>
		<Polyline
			IsVisible="{Binding $parent[ItemsControl].UltraSoundVisible, Mode=OneWay}"
			Opacity=".4"
			Fill="{Binding $parent[ItemsControl].UltraSoundBrush, Mode=OneWay}"
			Points="{Binding Points, Mode=OneWay}" />
		</DataTemplate>
	</ItemsControl.ItemTemplate>
</ItemsControl>
```
de valami miatt nem működik.

Reply:
Parent-ezés helyett próbáltad konstanssal? (Én személy szerint az nehezen látom át, pláne így este, ezért mindig konstansozok először.)

Reply:
Éredekes igy müködik:
```XAML
<ItemsControl Items="{Binding UltraSoundGeometries}">
							<ItemsControl.ItemTemplate>
								<DataTemplate>
									<Polyline
										IsVisible="{Binding UltraSoundVisible, Mode=OneWay}"
										Opacity=".4"
										Fill="green"
										Points="{Binding Points, Mode=OneWay}" />
								</DataTemplate>
							</ItemsControl.ItemTemplate>
						</ItemsControl>
```
Igy nem:
```XAML
<ItemsControl Items="{Binding UltraSoundGeometries}">
							<ItemsControl.ItemTemplate>
								<DataTemplate>
									<Polyline
										IsVisible="{Binding UltraSoundVisible, Mode=OneWay}"
										Opacity=".4"
										Fill="{Binding UltraSoundBrush, Mode=OneWay}"
										Points="{Binding Points, Mode=OneWay}" />
								</DataTemplate>
							</ItemsControl.ItemTemplate>
						</ItemsControl>
```
A lényeg hogy a `UltraSoundVisible` és `UltraSoundBrush` ugyan ott van mégis az egyik jól bindolodik a másik nem 💢

Reply:
Konstansokkal működik, csak megint egymás alá pakolja.
![image](https://user-images.githubusercontent.com/17666549/95375743-56fc3d80-08e0-11eb-9b91-2796e482c8d5.png)

Reply:
Huha ha [ezt](https://github.com/SzFMV2020-Osz/AutomatedCar-B/pull/89#discussion_r501155551) a megközelítést használjuk akkor van egy másik probléma mégpedig hogy az egyes `Polyline`-okat egymás alá rakja és nem fedésben egymásra ahogy alapbol tette eddig

Reply:
Én most nem tudok nekiállni debuggolni, nem tudom mi a scope ennyi kódból, de a parent nekem alapból fura volt, azért mondtam, hogy nézzétek meg konstanssal.

Bizonyos értelemben azt sem értem, hogy a szenzor látómező színe miért kell, hogy a szenzorban legyen. Emlékszem, hogy egyszer beszéltünk erről, hogy inkább oda kéne rakni mint „már nem tudom hova”, de tulajdonképpen az a vizualizáció magánügye, hogy milyen színnel kerül megjelenítésre, nem a szenzor sajátossága. Akár a XAML-be is lehetne égetni mint a highlight színt. Azt tuti írtam nektek a minap.

Reply:
Ahogy azt már [korábban is beszéltétek](https://github.com/SzFMV2020-Osz/AutomatedCar-B/pull/78#issuecomment-704801714) a #78 Az Avalonia [Itemsrepeter](https://avaloniaui.net/docs/controls/itemsrepeater) hivatkozik az [UWP](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.itemsrepeater?view=winui-2.5-prerelease)-re.
Igen ám de itt a [Layout](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.itemsrepeater.layout?view=winui-2.5-prerelease#Microsoft_UI_Xaml_Controls_ItemsRepeater_Layout) van amit lehetne használni, mert ez default [StackLayout](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.stacklayout?view=winui-2.5-prerelease)-ot használ ami ugye az említett problémát okozza ezért ezt helyettesiteni kéne valami mással. Találtam is két másikat a UWP Docsban, a [FlowLayout](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.flowlayout?view=winui-2.5-prerelease)-ot és a [UniformGridLayout](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.uniformgridlayout?view=winui-2.5-prerelease). Nem ismerem ezeket de ezeknek van más olyan [propertyje](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.uniformgridlayout.minitemheight?view=winui-2.5-prerelease#Microsoft_UI_Xaml_Controls_UniformGridLayout_MinItemHeight) ami talán segíthet